eng/Versions.props

```diff
@@ -11,4 +11,10 @@
     <MicrosoftDotNetBuildTasksPackagingVersion>6.0.0-beta.21366.1</MicrosoftDotNetBuildTasksPackagingVersion>
     <MicrosoftDotNetBuildTasksTemplatingVersion>6.0.0-beta.21366.1</MicrosoftDotNetBuildTasksTemplatingVersion>
   </PropertyGroup>
+  <PropertyGroup>
+    <MicrosoftDotNetBuildTasksInstallersPackageVersion>6.0.0-beta.21366.1</MicrosoftDotNetBuildTasksInstallersPackageVersion>
+    <MicrosoftDotNetBuildTasksWorkloadsPackageVersion>6.0.0-beta.21366.1</MicrosoftDotNetBuildTasksWorkloadsPackageVersion>
```

Comment:
Do these need to be added as dependencies in `eng/Version.Details.xml`? if not, then they should use `$(MicrosoftDotNetBuildTasksPackagingVersion)` as the property value, if it's meant to use the same version.

Reply:
Yeah they should be added to Version.Details.xml, @directhex